pkgs/applications/graphics/apitrace/default.nix

```diff
@@ -2,11 +2,11 @@
 
 stdenv.mkDerivation rec {
   name = "apitrace-${version}";
-  version = "7.1";
+  version = "7.1-git-20170411";
```

Comment:
I think a ```git describe```-like version scheme works best for unstable releases of upstream projects that do releases. (I think the point about using date as version was for projects that didn't do releases at all.) Using ```pre``` assumes you know the next version number. Using ```git describe```-like doesn't assume anything.